torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4307,9 +4301,14 @@ def bound_sympy(self, expr: sympy.Expr, size_oblivious: bool = False) -> ValueRa
         var_to_range = {x: self.var_to_range.get(x, None) for x in expr.free_symbols}
         if size_oblivious:
             # Clamp values of size-like variables
+            # NB: discarding the old upper bound in intentional, per
+            # https://github.com/pytorch/pytorch/pull/123675
```

Comment:
I missed https://github.com/pytorch/pytorch/pull/123675, but I just want to say that, as you know, I really dislike it :)